core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -383,6 +392,36 @@ var TopicsAndSkillsDashboardPage = function() {
     await waitFor.pageToFullyLoad();
   };
 
+  this.unassignSkillWithIndex = async function(skillDescription, topicIndex) {
```

Comment:
Thanks for switching to using skill descriptions, but now the function name is confusing

Reply:
Fixed.